packages/aws-cdk/lib/init.ts

```diff
@@ -211,12 +212,12 @@ export async function printAvailableTemplates(language?: string) {
   }
 }
 
-async function initializeProject(template: InitTemplate, language: string, canUseNetwork: boolean) {
+async function initializeProject(template: InitTemplate, language: string, canUseNetwork: boolean, generateOnly: boolean) {
   await assertIsEmptyDirectory();
   print(`Applying project template ${colors.green(template.name)} for ${colors.blue(language)}`);
   await template.install(language, process.cwd());
   await initializeGitRepository();
```

Comment:
should not run `initializeGitRepository` if `generateOnly` is `true`

Reply:
Done